postprocessing/mprofile_20240605175646.dat

```diff
@@ -0,0 +1,23 @@
+FUNC __main__.main 227.7188 1717631807.5411 254.3594 1717631808.2755 0
```

Comment:
Delete this file, seems like accidentally committed from memory profiling.

Reply:
Sure.